[2024-01-29T01:24:52.090+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: summoner_id_dag.fetch_summoner_ids manual__2024-01-29T00:24:45.038932+00:00 [queued]>
[2024-01-29T01:24:52.095+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: summoner_id_dag.fetch_summoner_ids manual__2024-01-29T00:24:45.038932+00:00 [queued]>
[2024-01-29T01:24:52.095+0100] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-29T01:24:52.112+0100] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): fetch_summoner_ids> on 2024-01-29 00:24:45.038932+00:00
[2024-01-29T01:24:52.115+0100] {standard_task_runner.py:60} INFO - Started process 24952 to run task
[2024-01-29T01:24:52.119+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'summoner_id_dag', 'fetch_summoner_ids', 'manual__2024-01-29T00:24:45.038932+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/summoner_id_dag.py', '--cfg-path', '/tmp/tmpb5r8nf8_']
[2024-01-29T01:24:52.121+0100] {standard_task_runner.py:88} INFO - Job 58: Subtask fetch_summoner_ids
[2024-01-29T01:24:52.150+0100] {task_command.py:423} INFO - Running <TaskInstance: summoner_id_dag.fetch_summoner_ids manual__2024-01-29T00:24:45.038932+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-01-29T01:24:52.208+0100] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='irachide1@gmail.com' AIRFLOW_CTX_DAG_OWNER='hadoop' AIRFLOW_CTX_DAG_ID='summoner_id_dag' AIRFLOW_CTX_TASK_ID='fetch_summoner_ids' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:24:45.038932+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-29T00:24:45.038932+00:00'
[2024-01-29T01:24:52.212+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-29T01:24:52.214+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/hadoop/Desktop/projectBD/scripts/fetch_summoner_ids.py ']
[2024-01-29T01:24:52.223+0100] {subprocess.py:86} INFO - Output:
[2024-01-29T01:24:59.678+0100] {subprocess.py:93} INFO - Successfully uploaded /home/hadoop/Desktop/projectBD/scripts/../output/summoner_details.json to /user/hadoop/lol/raw
[2024-01-29T01:24:59.696+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-29T01:24:59.721+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=summoner_id_dag, task_id=fetch_summoner_ids, execution_date=20240129T002445, start_date=20240129T002452, end_date=20240129T002459
[2024-01-29T01:24:59.760+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-29T01:24:59.769+0100] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
